### PR TITLE
fix #131, threads multi selection

### LINKED ISSUE #131: multiselect operations in threads(conversations) are behaving strangely
Device Meizu Pro 5
channel RC 2019-W33/2

Operation: 
- Longpress a conversation to enable multiselect
- Select, then deselect, more then one thread in a place downward in the conversations list (away from immediate view)
- scroll back up, then down again to the same items

result is that the messages are again selected.

### PR DESCRIPTION
**Pull request by lduboeuf**
This fix the issue for multi thread selection.
As ListItem seems not to work well, lets try with ListitemWithAction like in Dialer-app.

### CODE REVIEW

File: src/qml/MainPage.qml [hunk 1]
```diff
@@ -74,7 +74,7 @@ Page {
         property alias leadingActions: leadingBar.actions
         property alias trailingActions: trailingBar.actions
 
-        title: i18n.tr("Messages")
+        title: mainPage.selectionMode ? i18n.tr("Select") : i18n.tr("Messages")
```
Comment (lduboeuf) [inline src/qml/MainPage.qml:77]: can't make "Select" displayed in my test case. Did i missed something ? Are there any build steps that i don't know ?

Comment (lduboeuf) [reply src/qml/MainPage.qml:77]: I thought there is a fallback to `en` when no translations found. The `.pot` is updated as it should be i guess.

**Comment by lduboeuf** [comment]
The UI is not that good. The red lines and grey background doesn't respect what there is actually. I don't know how to style ListItemWithAction yet
[Edfit] : seems not possible to overwrite easily the red border, background is modifiable with selectedColor property

**Comment by lduboeuf** [comment]
to check: Must be tested in desktop mode ( i can't )

**Comment by krille-chan** [comment]
Hm, I can't test in desktop mode too at the moment. Could you use ListItem from https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Components/ListItem.html?highlight=listitem instead of ListItemWithAction? ListItem is quiet well themeable.

**Comment by lduboeuf** [comment]
i replaced ListItem because it was not working well for multi selection. i don't know why

**Comment by lduboeuf** [comment]
I will give again a try later.
 But that PR still fix the issue for multi selection which,  reported by someone is currently dangerous on stable

**Comment by lduboeuf** [comment]
I give up with ListItem, since it doesn"t work properly :-/

**Comment by Flohack74** [comment]
Hmm no help in App dev? Can we close this here then?

**Comment by lduboeuf** [comment]
humm haven't tried. But anyway, it works fine like this. Only we have a red border when selecting the thread as it is in dialer app or contact app. If it is a blocker then we can close this PR

**Comment by lduboeuf** [comment]
screenshot20191227_211733949
( I removed all texts in the screenshot )

**Comment by Flohack74** [comment]
@cibersheep what do you think?

**Comment by lduboeuf** [comment]
The border of ListItemWithAction from address-book-app is here: https://github.com/ubports/address-book-app/blob/xenial/src/imports/Ubuntu/Contacts/ListItemWithActions.qml#L268
We could make it a property

**Comment by cibersheep** [comment]
> @cibersheep what do you think?

If I understand correctly: is working but with the aesthetic issues, right? I would say go ahead with it while we fix the cosmetics afterwards.

Kind of in subject: Calculator has a similar issue with `ListItem`, I thought it was me

**Comment by cibersheep** [comment]
> We could make it a property

Thanks for the link. I can take a look afterwards... if I don't forget :P

**Comment by lduboeuf** [comment]
@cibersheep just done it here :): https://github.com/ubports/address-book-app/pull/107
Tried with "transparent" color, and feels cleaner ( i think ), for dialer-app and address-book-app too:

**Comment by cibersheep** [comment]
Oops... sorry sir, I didn't see that and I did that: https://github.com/ubports/address-book-app/pull/109

That takes out the border, background color and label color.

**Comment by Flohack74** [comment]
So @cibersheep wins or what =)

**Comment by cibersheep** [comment]
We all win :)

**Comment by lduboeuf** [comment]
Oh yes even better

**Pull request merged**